lxd/daemon.go

```diff
@@ -638,6 +639,13 @@ func (d *Daemon) createCmd(restAPI *mux.Router, version string, c APIEndpoint) {
 				return response.NotImplemented(nil)
 			}
 
+			if action.ReBACAccessHandler != nil {
+				resp := action.ReBACAccessHandler(d, r)
+				if resp != response.EmptySyncResponse {
+					return resp
+				}
+			}
+
```

Comment:
Does this mean if a `ReBACAccessHandler` is defined for the endpoint action it is always called, regardless of whether or not we have an openfga client configured?

Also, if ReBAC is configured, do we still need to call the standard `AccessHandler`?

I wonder if a better pattern would be to remove the `ReBACAccessHandler` and perform the check inside the `AccessHandler` instead if the client is set. We could have utils for building access handlers e.g. `access.NewHandler().Authenticated().AllowedReBACRelations("admin", "images_view").AllowedCandidRoles("admin", "images_view")` or similar.